content/en/docs/about/membership.md

```diff
@@ -0,0 +1,193 @@
++++
+title = "Community Membership"
+description = "Guidelines for contributing to Kubeflow"
+weight = 30
+aliases = ["/docs/membership/"]
++++
+
+
+**Note:** This document is a work in progress
+
+This document outlines the various responsibilities of contributor roles in Kubeflow. Kubeflow is divided into working groups that have stewardship over different subprojects/repositories
+
+Responsibilities for most roles are scoped to these repositories.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | Sponsored by 2 reviewers and multiple contributions to the project | Kubeflow GitHub org member|
+| Reviewer | Review contributions from other members | History of review and authorship in a repository | [OWNERS] file reviewer entry |
+| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to a repository | [OWNERS] file approver entry|
+| WG Lead  | Provides technical leadership for a Working Group | Have sufficient domain knowledge to provide effective technical leadership | [wgs.yaml] entry |
+| WG Chair | Provides overall leadership for a Working Group | Have sufficient domain knowledge to provide effective leadership | [wgs.yaml] entry |
+
+**Note:** Detail documentation for Working Group structure and responsibilities can be found at [wg-governance.md](https://github.com/kubeflow/community/blob/master/wgs/wg-governance.md)
+
+## New contributors
+
+[New contributors] should be welcomed to the community by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.
+
+## Established community members
+
+Established community members are expected to demonstrate their adherence to the principles in this document, familiarity with project organization, roles, policies, procedures, conventions, etc., and technical and/or writing ability. Role-specific expectations, responsibilities, and requirements are enumerated below.
+
+## Member
+
+Members are *[continuously active]* contributors in the community. They can have issues and PRs assigned to them and tests are automatically run for their PRs. Members are expected to remain active contributors to the community.
+
+**Defined by:** Member of the Kubeflow GitHub organization
+
+### Requirements
+
+- Enabled two-factor authentication on their GitHub account
+- Have made **at least** 2-3 [contributions] to the project or community.
```

Comment:
```suggestion
- Have made **at least** 2-3 [contributions] to the project or community, preferably related to more than a single working group.
```

Reply:
I wonder if this is unnecessarily "high" bar to entry. Some prospective members could be interested in specific subsets of the Kubeflow project.

Reply:
+1. I think for some areas people may choose to just stay in one domain (e.g., training). Probably not necessary to require this.

Reply:
ack, and agree